Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -21,56 +21,79 @@ extension Binding {
+            NavigationView {
+                List {
+                    Section {
+                        Picker("Audio Alert", selection: $confAudio.onChange(setAudio), content: {
+                            ForEach(audioFiles, id: \.self) {
+                                audioFile in Text(audioFile.description).tag(audioFile.rawValue)
+                            }
+                        })
+                    } header: {
+                        Text("Audio")
+                    }
+                    Section {
+                        Picker("Pattern", selection: $confLightPattern.onChange(setLights), content: {
+                            ForEach(lightPatterns, id: \.self) {
+                                lightPattern in Text(lightPattern.description).tag(lightPattern.rawValue)
+                            }
+                        })
+                        Picker("Brightness", selection: $confLightBrightness.onChange(setBrightness), content: {
+                            ForEach(lightBrightness, id: \.self) {
+                                brightnessLevel in Text(brightnessLevel.description).tag(brightnessLevel.rawValue)
+                            }
+                        })
+                        ColorPicker("Color", selection: $confLightColor.onChange(setColor), supportsOpacity: false
+                        )
+                    } header: {
+                        Text("Lights")
+                    }
+                    Section {
+                        NavigationLink(destination: RearRiderLogView()) {
+                            Text("View log")
                         }
-                    })
-                } header: {
-                    Text("Audio")
-                }
-                Section {
-                    Picker("Pattern", selection: $confLightPattern.onChange(setLights), content: {
-                        ForEach(lightPatterns, id: \.self) {
-                            lightPattern in Text(lightPattern.description).tag(lightPattern.rawValue)
-                        }
-                    })
-                    Picker("Brightness", selection: $confLightBrightness.onChange(setBrightness), content: {
-                        ForEach(lightBrightness, id: \.self) {
-                            brightnessLevel in Text(brightnessLevel.description).tag(brightnessLevel.rawValue)
-                        }
-                    })
-                    ColorPicker("Color", selection: $confLightColor.onChange(setColor), supportsOpacity: false
-                    )
-                } header: {
-                    Text("Lights")
+                    } header: {
+                        Text("Log")
+                    }
                 }
             }
+            .frame(maxHeight: 500)
+            
+            Button(action: saveConf) {
+                Text("Save")
+            }
+            .disabled(!configChanged)
```

Comment:
Okay I see why that boolean was introduced now. I don't think having a save button is necessary, though. We can have the conf auto save after every change, like it did earlier. It's more user friendly, and it's more in line with how other applications work. Is there a specific reason this functionality needed to be changed to this?

Reply:
The way I wrote the code to send the config over BT is that it's sending everything at once. If for example you keep playing with the settings it will send each time traffic over Bluetooth which it's not the worst but I thought we could be as efficient as possible.

Reply:
As a user, I'm going to want to see my config changes - i.e. see how the lights look when I make a change. So I'd be hitting that save button after every little change, essentially doing the exact thing you were trying to prevent.
 
We really don't need to be focusing on being efficient as possible as much as we need to focus on things working. But, this actually makes things less efficient since now there's more work to do the same thing we had before. This feels like a step backward to me.

Reply:
As a user, I'm going to want to see my config changes - i.e. see how the lights look when I make a change. So I'd be hitting that save button after every little change, essentially doing the exact thing you were trying to prevent.
 
We really don't need to be focusing on being efficient as possible as much as we need to focus on things working. But, this actually makes things less efficient since now there's more work to do the same thing we had before. This feels like a step backward to me.

Reply:
Ok I think we just had different ideas how to go about it. I can remove the save button and go back to saving every time a change is made.